### PR TITLE
feat: refactor export component

### PR DESCRIPTION
**Pull request by loido**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

**Review by ThinhKimVo — APPROVED** [review]
look good!

File: src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -160,7 +160,3 @@ const Accordion = React.forwardRef<FlatList, AccordionProps>(
     )
   },
 )
-
-Accordion.displayName = 'Accordion'
-
-export default React.memo(Accordion)
```
Comment (huydosgtech) [inline src/components/Accordion/Accordion.tsx:166]: should we keep React.memo ? 
We can use like below:
const MemoizedAccordion = React.memo(Accordion);
export { MemoizedAccordion };

File: src/components/Progress/Progress.tsx [hunk 2]
```diff
@@ -140,10 +140,6 @@ const Progress = forwardRef<View, IProgressProps>(
   },
 )
 
-Progress.displayName = 'Progress'
-
-export default memo(Progress)
-
```
Comment (huydosgtech) [inline src/components/Progress/Progress.tsx:146]: and here too

File: src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (loido) [reply src/components/Accordion/Accordion.tsx:166]: oke i'll keep memo at there. But i dont think the `MemoizedAccordion` is good. So i'll keep the name `Accordion`.
Do u agree with me?

Comment (huydosgtech) [reply src/components/Accordion/Accordion.tsx:166]: ok, it just example, you can change that name

**Comment by baonguyenhsts** [comment]
Could you please investigate why the github actions failed?

File: src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (loido) [reply src/components/Accordion/Accordion.tsx:166]: done

File: src/components/Progress/Progress.tsx (hunk 2 above)
Comment (loido) [reply src/components/Progress/Progress.tsx:146]: done

**Review by huydosgtech — APPROVED** [review]

**Pull request merged**